lib/matplotlib/axes/_axes.py

```diff
@@ -5875,10 +5875,10 @@ def table(self, **kwargs):
     #### Data analysis
 
     @_preprocess_data(replace_names=["x", 'weights'], label_namer="x")
-    def hist(self, x, bins=None, range=None, normed=False, weights=None,
+    def hist(self, x, bins=None, range=None, density=None, weights=None,
```

Comment:
I think this is safe to do.  If people are using positional it just works, if they are using keyword it still just works.  By putting this here we make it clearer which the preferred name is.